plugins/pluginmanagers/lazy.nix

```diff
@@ -1,61 +1,33 @@
 {
   lib,
   helpers,
-  config,
   pkgs,
   ...
 }:
 with lib;
```

Comment:
```suggestion

```

If we're in here migrating the plugin, we can clean up the `with lib;` usage, too.